Makefile

```diff
@@ -206,7 +205,7 @@ changelog:
 	$(shell cat $(TMPFILE) >> changelog.txt)
 	$(shell rm $(TMPFILE))
 
-install: .gopathok install.bin install.man install.cni install.systemd
+install: .gopathok install.bin install.man install.cni install.systemd install.python
```

Comment:
umm, can we remove install.python from here? Given that we intend to separate out the python components long term? Also, can we include install.completions here instead?